src/components/Select.jsx

```diff
@@ -342,6 +342,10 @@ export default class Select extends Component {
    * Pressing Escape will close the options area.
    */
   _onKeyDown (event) {
+    if (this.props.disabled) {
+      event.preventDefault();
+      return;
```

Comment:
If you return early here this whole block never will be called:

    if (this.props.wrapperProperties && this.props.wrapperProperties.onKeyDown) {
      this.props.wrapperProperties.onKeyDown(event);
    }

I think it's better to this:

    if(!this.props.disabled && filter(this.props.children, isOption).length > 0) {

Is there a specific reason why you called preventDefault?